### PR TITLE
Add support for setting position, normal, or texture coordinate individually for a vertex

### PR DESCRIPTION
**Pull request by tpietzsch**
Bonus: minor bugfix and javadoc fixes

### CODE REVIEW

File: src/main/java/net/imagej/mesh/Vertices.java
Comment (ctrueden) [inline src/main/java/net/imagej/mesh/Vertices.java:269]: Should we add a regression test?

**Review by ctrueden — APPROVED** [review]
All of this is fine as far as I'm concerned. But I defer to @kephale whether he likes it as well. And if so: please feel free to push the merge button!

**Comment by kephale** [comment]
Looks good to me

**Pull request merged**
